core/src/main/java/hudson/model/Queue.java

```diff
@@ -1567,8 +1567,16 @@ public void maintain() {
                 }
             }
 
-            if (s != null)
-                s.sortBuildableItems(buildables);
+            if (s != null) {
+                try {
+                    s.sortBuildableItems(buildables);
+                } except (Throwable e) {
+                    // JENKINS-52159 accelerated-build-now-plugin throws a ClassCastException
```

Comment:
A plugin with ~500 installs can't really motivate a core change like this. Either this makes sense to apply on its own as general robustness (in which case no need to mention the issue, or plugin), or this change isn't worth it.